src/Npgsql/TypeMapping/NpgsqlTypeMapping.cs

```diff
@@ -18,7 +18,7 @@ public class NpgsqlTypeMappingBuilder
         /// (schema.typename) - the latter can be used if you have two types with the same
         /// name in different schemas.
         /// </remarks>
-        public string? PgTypeName { get; set; }
+        public string PgTypeName { get; set; } = default!;
```

Comment:
Not sure this is really correct - this really is null by default, and it's the user's responsibility to populate it to non-null (which we verify in Build()).